app/views/layouts/_logo.html.erb

```diff
@@ -0,0 +1,9 @@
+<a href="/" class="site-logo" aria-label="<%= community_name %> Home">
+  <% if SiteConfig.logo_svg.present? %>
```

Comment:
Should we not be using just `if logo_svg.present?` here?

Reply:
Ah so logo svg seems to be this method in application_helper.rb:

```
  def logo_svg
    if SiteConfig.logo_svg.present?
      SiteConfig.logo_svg.html_safe # rubocop:disable Rails/OutputSafety
    else
      inline_svg_tag("devplain.svg", class: "logo", size: "20% * 20%", aria: true, title: "App logo")
    end
  end
```

Hence, I see why you used `SiteConfig.logo_svg.present?` instead of `logo_svg.present?` for this if statement.
However, then it would seem like the else statement in logo_svg is useless as we never want it to render devplain, but rather the text. Perhaps you can check and we can remove it? 

In that case also, perhaps we wouldn't need to pass logo_svg all around, but we could just use the SiteConfig variable if we get rid of the else.

Reply:
Um, I guess so. Fixing :)

Reply:
Re application_helper.rb - that's a good catch! I was not aware of this `if` statement in logo_svg. I think you're 100% right - we should drop it. But I believe that means refactoring some other bits where we use it, including all the logic that makes the logo_svg required. And THAT in particular is actually something I don't wanna touch because it's back-end yada yada yada... But this is exactly what we wanna handle separately: https://github.com/forem/InternalProjectPlanning/issues/83

Does that make sense?

Reply:
Coolios, happy with it being removed and the code cleaned up in a follow up PR 👍